### PR TITLE
Added Gyroscope Blockhandler Support

### PR DESCRIPTION
**Pull request by MerlinofMines**
This commit adds decent Gyroscope Blockhandler support with the ability to control all Gyroscope specific properties.

This commit implements #65

### CODE REVIEW

File: EasyCommands.Tests/ScriptTests/GyroscopeBlockTests.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+using Sandbox.ModAPI.Ingame;
+using VRageMath;
+
+namespace EasyCommands.Tests.ScriptTests {
+    [TestClass]
+    public class GyroscopeBlockTests {
+        [TestMethod]
+        public void EnableGyroscopeOverrides() {
+            String script = @"
+set the ""test gyro"" auto to false
```
Comment (pandincus) [inline EasyCommands.Tests/ScriptTests/GyroscopeBlockTests.cs:13]: maaaayyyybeee think about just calling it override and having it map 1:1 to the boolean.

File: EasyCommands/BlockHandlers/GyroscopeBlockHandlers.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+using VRage.Game.ModAPI.Ingame.Utilities;
+using VRage.Game.ObjectBuilders.Definitions;
+using VRageMath;
+
+namespace IngameScript {
+    partial class Program {
+        public class GyroscopeBlockHandler : FunctionalBlockHandler<IMyGyro> {
+            public GyroscopeBlockHandler() {
+                AddBooleanHandler(PropertyType.AUTO, b => !b.GyroOverride, (b, v) => b.GyroOverride = !v);
+                AddNumericHandler(PropertyType.RANGE, b => b.GyroPower, (b, v) => b.GyroPower = v, 0.1f);//TODO: Check this
```
Comment (pandincus) [inline EasyCommands/BlockHandlers/GyroscopeBlockHandlers.cs:25]: check this ;-)

**Review by pandincus — APPROVED** [review]

File: EasyCommands/BlockHandlers/GyroscopeBlockHandlers.cs (hunk 2 above)
Comment (MerlinofMines) [reply EasyCommands/BlockHandlers/GyroscopeBlockHandlers.cs:25]: checked

File: EasyCommands.Tests/ScriptTests/GyroscopeBlockTests.cs (hunk 1 above)
Comment (MerlinofMines) [reply EasyCommands.Tests/ScriptTests/GyroscopeBlockTests.cs:13]: did it

**Pull request merged**